dev/io.openliberty.concurrent.internal.cdi/resources/io/openliberty/concurrent/internal/cdi/resources/CWWKCMessages.nlsprops

```diff
@@ -0,0 +1,41 @@
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#CMVCPATHNAME io.openliberty.concurrent.internal.cdi/resources/io/openliberty/concurrent/internal/cdi/resources/CWWKCMessages.nlsprops
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKC
+#COMPONENTNAMEFOR CWWKC Jakarta EE Concurrency
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# All messages must use the range CWWCK1400 to CWWCK1499
+
+CWWKC1400.unsupported.return.type=CWWKC1400E: {0} is not a valid return type for the {1} method of the {2} class. Valid return types for methods that are annotated with {3} are: {4}.
+CWWKC1400.unsupported.return.type.explanation=The Jakarta Concurrency specification limits the range of valid return types for methods that have this annotation.
+CWWKC1400.unsupported.return.type.useraction=Update the method within the application to switch to one of the supported return types or remove the annotation from it.
+
+CWWKC1401.class.anno.disallowed=CWWKC1401E: The {0} annotation is only permitted at the method level. It cannot be used at the class level on the {1} class.
+CWWKC1401.class.anno.disallowed.explanation=This annotation must not be specified at the class level.
+CWWKC1401.class.anno.disallowed.useraction=Move the annotation to the individual methods of the class.
+
+CWWKC1402.not.managed.executor=CWWKC1402E: The {0} annotation on the {1} method of the {2} class specifies the {3} JNDI name, which resolves to the {4} resource which is not a ManagedExecutorService or ManagedScheduledExecutorService. Interfaces implemented by the resource are: {5}.
+CWWKC1402.not.managed.executor.explanation=The JNDI name must resolve to a ManagedExecutorService or ManagedScheduledExecutorService.
+CWWKC1402.not.managed.executor.useraction=Correct the JNDI name to specify a ManagedExecutorService or ManagedScheduledExecutorService that is accessible to the application component.
+
+CWWKC1403.unsupported.tx.type=CWWKC1403E: The {0} annotation specifies the {1} value, which is not permitted in combination with the {2} annotation that is on the {3} method of the {4} class. Permitted values are: {5}.
+CWWKC1403.unsupported.tx.type.explanation=Whether or not the annotations can be used in combination depends on the value that is specified.
+CWWKC1403.unsupported.tx.type.useraction=Update the application to switch to one of the supported values or remove one of the annotations.
```

Comment:
Reviewed Messages:
CWWKC1403.unsupported.tx.type=CWWKC1403E: The {0} annotation specifies the {1} value, which is not permitted in combination with the {2} annotation that is on the {3} method of the {4} class. Permitted values are: {5}.

CWWKC1403.unsupported.tx.type.explanation=The use of annotations in combination depends on the value that is specified.

CWWKC1403.unsupported.tx.type.useraction=Update the application to switch to one of the supported values or remove one of the annotations.

Reply:
That sounds awkward to me, but I'll update it as you recommend.

Reply:
CWWKC1403.unsupported.tx.type.explanation=The use of two annotations together depends on whether the value of the first annotation that is specified is permitted.

Reply:
Updating as requested.